src/glyph.rs

```diff
@@ -0,0 +1,141 @@
+use glib::translate::*;
+use pango_sys;
+use GlyphItem;
+use Item;
+use GlyphString;
+
+impl GlyphString {
+
+    pub fn num_glyphs(&self) -> i32 {
```

Comment:
Can this ever be negative? If not, `usize` or at least `u32` would be more useful. If it can be negative then the code below needs some checks

Reply:
Technically it can, not sure if it ever will be.
Added a check [here](https://github.com/gtk-rs/pango/pull/163/files#diff-6fed5ad9e758a0639326a7d2bd1cdb9aR14), not sure if that's what you had in mind